taskcluster/ci/beetmover-apt/kind.yml

```diff
@@ -0,0 +1,30 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+loader: taskgraph.loader.transform:loader
+
+transforms:
+    - taskgraph.transforms.from_deps
+    - mozillavpn_taskgraph.transforms.beetmover_apt
+    - taskgraph.transforms.task:transforms
+
+kind-dependencies:
+    - beetmover
+
+tasks:
+    push-to-gcr:
+        from-deps:
+            with-attributes:
+                build-type:
+                    - linux64/release-deb
+        run-on-tasks-for: [action]
```

Comment:
Just noting that this is a no-op as actions don't use the target tasks method that processes this key, but it's ok to leave it as if nothing else, it documents that we don't expect this task to run anywhere other than actions.

Reply:
If we want to remove this we should probably do everything, this is the same as []